app/src/main/java/org/mozilla/fenix/home/recenttabs/view/RecentTabs.kt

```diff
@@ -116,6 +116,7 @@ private fun RecentTabItem(
             }
         }
     }
+    Spacer(modifier = Modifier.width(16.dp))
```

Comment:
I think you are looking to increase the spacing between the header of "Recently saved " and not between each RecentTabItem (media, current tab). This change will in fact change increase the 8dp spacing to 24 (8+16) between each recent tab item (media, current open tab, search groups) that is provided by `verticalArrangement = Arrangement.spacedBy(8.dp)`. If you did want to increase the spacing between the recent tab and media to 24dp. You can change the spacedBy. 

I think the best way is to add a top margin of 16dp in the headers instead.

Reply:
This is modifying the area in between each recentTabItem, not the space between the headers. Should I add that `verticalArragment` to `Card` instead of the `Spacer`?

Reply:
I am not seeing where we want this additional 16dp spacer in between the tabs. The current 8dp you see in the screenshot is handled by the `verticalArrangement = Arrangement.spacedBy(8.dp)` in the `Column` of `RecentTabs`. 

<img width="355" alt="Screen Shot 2021-09-22 at 3 23 30 PM" src="https://user-images.githubusercontent.com/1190888/134410565-93e872f6-c6c0-4167-a840-74cd060246e2.png">

Reply:
Ahh it was those pesky ios designs that have 16dp in between!